### PR TITLE
[DRAFT] Add monitoring

### PR DESCRIPTION
**Pull request by alevani**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - "9090:9090"
+    networks:
+      - main
+
+  grafana:
+    image: grafana/grafana:4.5.2
+    ports:
+      - "3000:3000"  
+    networks:
+      - main
```
Comment (alevani) [inline docker-compose.yaml]: I need to add LR

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  main:
+    name: itu-minitwit-network
+
+
+services:
+  minitwit:
+    image: freakency/devops
+    container_name: minitwit
+    volumes:
+      - /tmp/db:/var/www/db
```
Comment (alevani) [inline docker-compose.yaml]: This need to be changed to live setup

File: docker-compose.yaml (hunk 1 above)
Comment (janschill) [inline docker-compose.yaml]: missing EOL ;)

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+prometheus_client==0.7.1
```
Comment (janschill) [inline requirements.txt]: this is not needed? Isn't this for Python?

**Comment by janschill** [comment]
What's up with all the merges in your commits?

File: requirements.txt (hunk 3 above)
Comment (alevani) [reply requirements.txt]: I was not sure about all that requirement file setup, it contained like 20 lines before I removed everything. I can't tell you if this is needed or not yet, let's try to figure that out later 🚀

Comment (janschill) [reply requirements.txt]: I am 93% sure, it is for Python dependency versioning.

**Pull request closed without merging**